### PR TITLE
docs: add getting started with abstract working of website

### PR DESCRIPTION
**Pull request by majordwarf**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Getting Started
+
+The module website development environment can be installed on macOS, Windows, and Linux (Debian or Ubuntu recommended). You’ll need at least 1GB of available RAM.
+
+---
+
+#### Setting up development workspace
+
+Start by cloning your fork of the Zulip repository and connecting the Zulip upstream repository:
```
Comment (jdrueckert) [inline docs/getting-started.md]: Zulip?

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Getting Started
```
Comment (jdrueckert) [inline docs/getting-started.md]: "Getting Started" as in local workspace setup (if not using the gitpod environment) for contributing should be on the main README. this additional doc here should only cover project structure details that would be too in-depth for the readme.

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+#### Docker Setup
+
+> **TODO**
+
+#### Manual Setup
+
+<details>
+  <summary>Steps</summary>
+
+1. Install the follwing dependencies on your system
```
Comment (jdrueckert) [inline docs/getting-started.md]:
```suggestion
1. Install the following dependencies on your system 

```

**Review by jdrueckert — CHANGES_REQUESTED** [review]

File: docs/getting-started.md (hunk 1 above)
Comment (majordwarf) [reply docs/getting-started.md]: Fixed it!

File: docs/getting-started.md (hunk 2 above)
Comment (majordwarf) [reply docs/getting-started.md]: Moved the whole setup documentation to README and renamed `getting-started.md` to `project-structure.md`.

File: docs/getting-started.md (hunk 3 above)
Comment (majordwarf) [reply docs/getting-started.md]: 👍

File: README.md [hunk 4]
```diff
@@ -24,7 +24,13 @@ Terasology’s basic engine can be extended by a huge amount (201 right now!) of
...
 ---
 
-<h2 id="built-with">Built With</h2>
+<h2> Getting Started </h2>
+
+The module website development environment can be installed on macOS, Windows, and Linux (Debian or Ubuntu recommended). You’ll need at least 1GB of available RAM.
+
+---
+
+<h2 id="built-with">Built Sith</h2>
```
Comment (jdrueckert) [inline README.md]:
```suggestion
<h2 id="built-with">Built With</h2>
```

File: docs/project-structure.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+├── blog            - Markdown files for the blog posts
+├── data            - Meta data for the website
+├── modules         - Markdown files for the modules data
+├── scripts         - Scripts to deploy the website to GitHub and Render
+├── src             - Most of the website's actual code as explained ↓
+│   ├── components  - React components we see throughout the website¹
+│   ├── layout      - The structure of the website, may be interpreted as a skeleton which is later filled with react components to bring it to life
+│   ├── pages       - Gatsby uses the pages directory to generate all the pages we see throughout the website²
+│   ├── template    - Not all pages needs to hard-coded, Gatsby uses templates to programmatically generate pages ³
+├── satic           - Images, Logos, robots.txt etc
```
Comment (jdrueckert) [inline docs/project-structure.md]:
```suggestion
├── static          - Images, Logos, robots.txt etc

```

**Review by jdrueckert — CHANGES_REQUESTED** [review]
Do you intend to add information for the "TODO" section within the context of this PR? If not, please remove the sections and add issues to add respective information.

**Pull request merged**